var/spack/repos/builtin/packages/r-genefilter/package.py

```diff
@@ -7,13 +7,13 @@
 
 
 class RGenefilter(RPackage):
-    """genefilter: methods for filtering genes from high-throughput experiments
+    """: methods for filtering genes from high-throughput experiments.
```

Comment:
I would undo this removal of "genefilter"

Reply:
Yep, I meant to remove those but missed this one.